src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -2502,6 +2535,48 @@ private static void RemoveLastNullCompletionResult(List<CompletionResult> result
             }
         }
 
+        private static void NativeCompletionScope(CompletionContext context, List<CompletionResult> result)
+        {
+            var wordToComplete = context.WordToComplete;
+
+            var typeResults = CompleteScope(wordToComplete, context);
+            if (typeResults != null)
+            {
+                result.AddRange(typeResults);
+            }
+
+            result.Add(CompletionResult.Null);
+        }
+
+        internal static List<CompletionResult> CompleteScope(string wordToComplete, CompletionContext context)
+        {
+            var results = new List<CompletionResult>();
+            var scopes = new List<string>() {"Global", "Local", "Script"};
```

Comment:
This could be a static array.

Reply:
The code has been modified to reflect this!